README.md

```diff
@@ -121,10 +121,205 @@ For example:
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:logs:_:_:log-group:RDSOSMetrics:log-stream:_"
+        },
+        {
+            "Action": [
+                "rds:DescribeDBParameters"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:rds:_:_:pg:_"
+        },
+        {
+            "Action": [
+                "rds:DescribeDBInstances",
+            "rds:DownloadDBLogFilePortion",
+            "rds:DescribeDBLogFiles"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:rds:_:_:db:_"
+        },
+        {
+            "Action": [
+                "rds:DescribeDBClusters"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:rds:_:_:cluster:\*"
+        }
+    ]
+}
+```
+
+Then, run this command from the CLI:
+
+```bash
+aws iam create-policy
+--policy-name autodba
+--policy-document file://autodba_policy.json
+--description "Allow AutoDBA to access RDS"
+```
+
+##### Create IAM role
+
+First, run this command from the CLI:
+
+> **Tip:** replace `JSON_HERE` in the `aws` command below with the following JSON.
+
+```json
+{"Version": "2012-10-17", "Statement": [{"Effect": "Allow", "Principal": {"Service": "ec2.amazonaws.com"}, "Action": "sts:AssumeRole"}]}
+```
+
+```bash
+aws iam create-role
+--role-name autodba
+--description "autodba collector"
+--assume-role-policy-document JSON_HERE
+```
+
+Then, run this command after replacing `AWS_ACCOUNT_ID` with your AWS Account ID:
+
+```
+aws iam attach-role-policy
+--role-name autodba
+--policy-arn arn:aws:iam::AWS_ACCOUNT_ID:policy/autodba
+```
+
+##### Attach IAM role to EC2 instance
+
+Either start a new `t3.small` EC2 instance and attach the IAM role during creation or attach the IAM role to an existing instance with the following command:
+
+```bash
+aws ec2 associate-iam-instance-profile
+--instance-id INSTANCE_ID
+--iam-instance-profile Name=autodba
+```
+
+
+#### Google Cloud Platform (GCP)
+
+##### Create Service Account
+
+Create a new service account for AutoDBA with the following command:
+
+```bash
+gcloud iam service-accounts create autodba --display-name "AutoDBA"
+```
+
+##### Add Roles to the Service Account
+
+Add the following roles to the service account:
+
+- Cloud SQL Viewer
+- Monitoring Service Agent
+- Pub/Sub Subscriber
+
+Add the roles with the following commands, replacing `PROJECT_ID`:
+
+```bash
+gcloud projects add-iam-policy-binding PROJECT_ID \
+    --member="serviceAccount:autodba@PROJECT_ID.iam.gserviceaccount.com" \
+    --role="roles/cloudsql.viewer"
+```
+
+```bash
+gcloud projects add-iam-policy-binding PROJECT_ID \
+    --member="serviceAccount:autodba@PROJECT_ID.iam.gserviceaccount.com" \
+    --role="roles/monitoring.notificationServiceAgent"
+```
+
+```bash
+gcloud projects add-iam-policy-binding PROJECT_ID \
+    --member="serviceAccount:autodba@PROJECT_ID.iam.gserviceaccount.com" \
+    --role="roles/pubsub.subscriber"
+```
```

Comment:
IMO, it's better to combine them all into a single code-block.

Reply:
fixed